pkg/api/api.go

```diff
@@ -97,6 +113,7 @@ type API struct {
 	sourceIPs            *middleware.SourceIPExtractor
 	indexPage            *IndexPageContent
 	HTTPHeaderMiddleware *HTTPHeaderMiddleware
+	CORSOrigin           *regexp.Regexp
```

Comment:
It is a small thing. But let's change `CORSOrigin` to `corsOrigin`. We don't want to expose this field

Reply:
updated